webapp/tests/test_functions.py

```diff
@@ -3864,3 +3862,138 @@ def test_summarize_1minute_alignToFrom(self):
               },
               seriesList, "1minute", func, True)
           self.assertEqual(result, expectedResults[func])
+
+    def test_exponentialMovingAverage_integerWindowSize(self):
```

Comment:
There is quite a bit of duplicated code in these tests ? Could you try refactoring that ?

Having a helper function called in the various tests and putting some of the values (like the dates) into global constants might help